src/EventStore.Core.Tests/Services/Transport/Grpc/StreamsTests/DeleteTests.cs

```diff
@@ -0,0 +1,151 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using EventStore.Client.Streams;
+using EventStore.Core.Data;
+using EventStore.Core.Services.Transport.Grpc;
+using Google.Protobuf;
+using Grpc.Core;
+using NUnit.Framework;
+
+namespace EventStore.Core.Tests.Services.Transport.Grpc.StreamsTests {
+	[TestFixture]
+	public class DeleteTests {
+		private const string StreamName = nameof(DeleteTests);
+
+		public abstract class DeleteExistingStreamSpecification<TLogFormat, TStreamId>
+			: GrpcSpecification<TLogFormat, TStreamId> {
+			private readonly DeleteReq.Types.Options _options;
+			private DeleteResp _deleteResponse;
```

Comment:
is is assigned to but not checked, is it supposed to be? same for _appendResponse

Reply:
(also in tombstone tests)